q_yaar_platform/q_yaar/settings.py

```diff
@@ -322,3 +332,8 @@
     },
     "loggers": {**DJANGO_DEFAULT_LOGGERS, **APPLICATION_LOGGERS},
 }
+
+#######################################################################################################################
+
+# Game Config
+GAME_CODE_LENGTH = config("GAME_CODE_LENGTH", cast=int, default=6)
```

Comment:
Keeping things configurable. We should be able to change business rules without writing any code.